src/packet.c

```diff
@@ -1,3 +1,819 @@
+                                            BCMPDFUEndMessage,
+                                            //TODO: set size and callbacks
+                                            1,
+                                            false,
+                                            NULL,
+                                            NULL,
+                                        },
+                                        {
+                                            BCMPDFUAckMessage,
+                                            //TODO: set size and callbacks
+                                            1,
+                                            false,
+                                            NULL,
+                                            NULL,
+                                        },
+                                        {
+                                            BCMPDFUAbortMessage,
+                                            //TODO: set size and callbacks
+                                            1,
+                                            false,
+                                            NULL,
+                                            NULL,
+                                        },
+                                        {
+                                            BCMPDFUHeartbeatMessage,
+                                            //TODO: set size and callbacks
+                                            1,
+                                            false,
+                                            NULL,
+                                            NULL,
+                                        },
+                                        {
+                                            BCMPDFURebootReqMessage,
+                                            //TODO: set size and callbacks
+                                            1,
+                                            false,
+                                            NULL,
+                                            NULL,
+                                        },
+                                        {
+                                            BCMPDFURebootMessage,
+                                            //TODO: set size and callbacks
+                                            1,
+                                            false,
+                                            NULL,
+                                            NULL,
+                                        },
+                                        {
+                                            BCMPDFUBootCompleteMessage,
+                                            //TODO: set size and callbacks
+                                            1,
+                                            false,
+                                            NULL,
+                                            NULL,
+                                        }};
```

Comment:
I also really hate how this is formatted? It seems like it would look a lot prettied one line at a time:

```
static struct SerialLUT PACKET_LUT[] = {
{ BCMPHeartbeatMessage, sizeof(BCMPHeartbeat), false, NULL, NULL, },
{ BCMPEchoRequestMessage, sizeof(BCMPEchoRequest), false, NULL, NULL,},
...
```

Reply:
Yep, I don't hate it as much, but I totally understand. Let's stick with standard clang-format for now. In a future PR if you can find a change to the clang-format rules that you like better, I think that can be totally fine.

Reply:
Sounds good!

Reply:
Tis now a bit prettier without touching the clang formatting.